<!-- -->
<launch>

<!-- 另起一个终端  保存栅格占用地图 -->
<node pkg="map_server" name="map_server" type="map_saver"   args="-f /home/liunao/map/region2-1_V2_chedi/region4-3_3rd_chedi/ori "  output="screen"/>
<!-- rosrun map_server map_saver -->

<node pkg="pcd2pgm" name="pcd2pgm" type="pcd2pgm" output="screen">

<!-- pcd文件名称-->
<!-- <param name="file_name" value= "/home/map/region4-4/1.pcd" /> -->
<!-- <param name="file_name" value= "/home/map/region4-3/box.pcd" /> -->
<param name="file_name" value= "/home/liunao/map/region2-1_V2_chedi/region4-3_3rd_chedi/saveMap_ori_horizontal_cut_normal.pcd" />
<!-- <param name="file_name" value= "/opt/csg/slam/navs/livos/result/xn/short/faster_lio_test_pcd_30_cm.pcd" /> -->
<!-- <param name="file_name" value= "/home/test_h.pcd" /> -->

<param name="png_file" value= "/home/test" />
<param name="auto_rotation" value= "0" />


<!-- 选取的范围　最小的高度-->
<param name="thre_z_min" value= "1.20" />
<!-- 选取的范围　最大的高度-->
<param name="thre_z_max" value= "2.5" />
<!--0 选取高度范围内的，１选取高度范围外的-->
<param name="flag_pass_through" value= "0" />
<!-- 半径滤波的半径-->
<param name="thre_radius" value= "0.5" />
<!-- 半径滤波的要求点数个数-->
<param name="thres_point_count" value= "1" />
<!-- 存储的栅格map的分辨率-->
<param name="map_resolution" value= "0.1" />
<!-- 转换后发布的二维地图的topic，默认使用map即可，可使用map_server保存-->
<param name="map_topic_name" value= "map" />
</node>

</launch>
